show/main.py

```diff
@@ -1224,12 +1224,23 @@ def users(verbose):
 
 @cli.command()
 @click.option('--since', required=False, help="Collect logs and core files since given date")
+@click.option('-g', '--global-timeout', default=30, type=int, help="Global timeout for techsupport in minutes. Default 30 mins")
```

Comment:
for the next two lines, how about:
```
@click.option('
    -g', '--global-timeout', default=30, type=int,
    help="Global timeout for techsupport in minutes. Default 30 mins"
)
```

Reply:
done

Reply:
I modified the help string to keep the line width <= 120 chars